pkg/release/helm_actions.go

```diff
@@ -88,13 +88,13 @@ func (rel *config) newUpgrade() *action.Upgrade {
 func (rel *config) newUninstall() *action.Uninstall {
 	client := action.NewUninstall(rel.Cfg())
 
-	client.KeepHistory = false                // TODO: pass it via flags
-	client.DeletionPropagation = "background" // TODO: pass it via flags
```

Comment:
Left it like this on purpose because helm itself assigns a default value, this is done to ensure if in any future version helm changes the default value then that is respected as well.

```
func parseCascadingFlag(cfg *Configuration, cascadingFlag string) v1.DeletionPropagation {
	switch cascadingFlag {
	case "orphan":
		return v1.DeletePropagationOrphan
	case "foreground":
		return v1.DeletePropagationForeground
	case "background":
		return v1.DeletePropagationBackground
	default:
		cfg.Log("uninstall: given cascade value: %s, defaulting to delete propagation background", cascadingFlag)
		return v1.DeletePropagationBackground
	}
}
```

Reply:
I can still assign a default value if needed

Reply:
@r3nic1e FYI

Reply:
didn't know it work like that, cool